modules/cluster-logging-updating-logging.adoc

```diff
@@ -61,38 +63,38 @@ by Red Hat, Inc
 
 .. Click *Subscription* -> *Channel*.
 
-.. In the *Change Subscription Update Channel* window, select *4.4* and click *Save*.
+.. In the *Change Subscription Update Channel* window, select *4.5* and click *Save*.
 
 .. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
 +
-The Cluster Logging Operator is shown as 4.4. For example:
+The Cluster Logging Operator is shown as 4.5. For example:
 +
 ----
 Cluster Logging
-4.4.0-201909201915 provided
+4.5.0-202005201915 provided
 by Red Hat, Inc
 ----
 
 . Check the logging components:
 
-.. Ensure that the Elasticsearch Pods are using a 4.4 image:
+.. Ensure that the Elasticsearch Pods are using a 4.5 image:
```

Comment:
Regarding this section and checking the images.  There is nothing about them that identifies they are 4.4. vs 4.5 because we now use a sha instead of the release version.  We might as well remove this all together.  Seems that we need to trust the new operator is giving us what we require and if we truly want to verify it "worked" we can:

* look at the age of the pods, they should be relatively new
* we would have to formulate a command to go inspect the images for the proper labels added when they were built